### PR TITLE
Remove mxdatetime

### PR DESCRIPTION
**Pull request by jeanconn**
Remove mxdatetime

### CODE REVIEW

File: make_esaview_data.py [hunk 1]
```diff
@@ -138,9 +139,9 @@ def get_intervals(start, nweeks):
     time for weekly loads.  Only return intervals for weeks which do not already
     have an output file (i.e. don't re-compute existing weeks).
     """
-    monday = start - start.mxDateTime.day_of_week  # Monday=0 .. Sunday=6
+    monday = start - (date.fromtimestamp(start.unix).isoweekday() - 1)  # isoweekday Monday=1 .. Sunday=7
```
Comment (taldcroft) [inline make_esaview_data.py]: Can you use `date.weekday()` which goes from 0 .. 6?

**Comment by taldcroft** [comment]
What did you do for testing?

**Comment by jeanconn** [comment]
I compared the start and stop days of the first returned interval using the
method and confirmed that the time stamps were the same after the change.

**Comment by taldcroft** [comment]
Testing sounds good then.

**Comment by jeanconn** [comment]
I'll update to use datetime.date.weekday and copy paste manual tests outputs into this PR for the record.

**Comment by jeanconn** [comment]
output of this routine from b72dd7c51f280f5246fb08bbc790961e6cc31b67 vs c7aa6b8:

```
i1 = make_esaview_data.get_intervals(DateTime(), 2)
In [6]: i1[0][0].date
Out[6]: '2016:038:00:00:00.000'
In [7]: i1[0][1].date
Out[7]: '2016:047:00:00:00.000'
In [8]: i1[1][0].date
Out[8]: '2016:045:00:00:00.000'
In [9]: i1[1][1].date
Out[9]: '2016:054:00:00:00.000'

i2 = make_esaview_data.get_intervals(DateTime(), 2)
In [7]: i2[0][0].date
Out[7]: '2016:038:00:00:00.000'
In [8]: i2[0][1].date
Out[8]: '2016:047:00:00:00.000'
In [9]: i2[1][0].date
Out[9]: '2016:045:00:00:00.000'
In [10]: i2[1][1].date
Out[10]: '2016:054:00:00:00.000'
```

**Comment by taldcroft** [comment]
:+1: for merge and deploy (HEAD and GRETA).

**Comment by jeanconn** [comment]
Before wrapping this up, does this need any kind of version update or some kind of tracking?

**Comment by taldcroft** [comment]
Nah.

**Comment by jeanconn** [comment]
Cool.  In general, I still have the slight problem trying to track changes in data/share, but if I update it correctly on GRETA and HEAD I don't see how it could surprise us later.

**Comment by taldcroft** [comment]
At least for share the right answer is using package entry points. For data the answer isn't always so clear, but at least some of the time it can work to have package data.

**Comment by jeanconn** [comment]
Installed on GRETA and HEAD.

**Pull request merged**